lib/src/main/java/com/thenewboston/data/dto/bankapi/blockdto/Blocks.kt

```diff
@@ -7,26 +7,40 @@ import kotlinx.serialization.Serializable
 
 @Serializable
 data class BlockList(
+
+    @SerialName("count")
     val count: Long,
+
+    @SerialName("next")
     val next: String?,
+
+    @SerialName("previous")
     val previous: String?,
-    val results: List<Block>
+
+    @SerialName("results")
+    val blocks: List<Block>
```

Comment:
Should we not keep it as `results`? It would be confusing calling `blocks.blocks `

Reply:
I saw the same in `BankList` where it's called `banks` and I liked the more semantic name. I think it's rather that the outer variable should be called `blockList` or just `response`, making it `response.blocks`.

Reply:
I like `response` sounds better. I suppose we tackle this as part of https://github.com/thenewboston-developers/Kotlin-SDK/issues/86